test/cases/testdata/jsonbuiltins/test-json-marshal-indent.yaml

```
@@ -0,0 +1,36 @@
+---
+cases:
+  - data: {}
+    modules:
+      - |
+        package test
+
+        p = x {
+          json.marshal_indent([1234500000 + 67890, 1e6 * 2, 1e109 / 1e100], "  ", x)
```

This should be possible to use the assignment operator for these tests like this:

```suggestion
          x := json.marshal_indent([1234500000 + 67890, 1e6 * 2, 1e109 / 1e100], "  ")
```

This would be more idiomatic modern rego.